### ISSUE TITLE
Unbreakable: false duplicates NBT tag for items spawned with {Unbreakable:1b}

### ISSUE THREAD

**Issue by Partonetrain** [nvb-uy/CIA#3]
It does set the unbreakable tag to false, but I would expect it to do nothing, or at least set the unbreakable tag without duplicating it into the "tag" tag.
My use-case here is I have a unbreakable version of an item as loot, but I want the normal (and loot) version of it to have custom attributes. Aside from this issue, I would also like to propose the option to exclude items that already have overrides via NBT. If not, that's okay, I will mixin [here](https://github.com/nvb-uy/CIA/blob/main/src/main/java/elocindev/customitemattributes/builder/AttributeBuilderCallback.java#L26-L29) to disable the unbreakable code.

To reproduce:

set an item in config to `unbreakable: false`, eg. 
```
{
      "item": "minecraft:diamond_sword",
      "overrides_main_hand": [
      ],
      "overrides_off_hand": [
      ],
      "overrides_head": [],
      "overrides_chest": [],
      "overrides_legs": [],
      "overrides_feet": [],
      "unbreakable": false
    },
```
give an item with the unbreakable tag, e.g. `/give @p diamond_sword{Unbreakable:1b}`
use `/data get entity @p SelectedItem` with said item in selected hotbar slot (or /ct hand vanilla if you have CraftTweaker)
Observe the data being duplicated into the "tag" tag: 
![image](https://github.com/nvb-uy/CIA/assets/19861808/f84044aa-a79c-48f4-af5f-bcb9270d0426)

**Comment by nvb-uy** [comment]
Yup, it's a known issue that will be fixed soon!

**Comment by nvb-uy** [comment]
Fixed on 2.0.0, please read full changelog as the config moved locations and the format also changed.
